Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/security.yml

```diff
@@ -0,0 +1,10 @@
+- type: entity
```

Comment:
shouldnt exist, you lose a rocket

Reply:
Use a table and give it 20% probability for extra rocket.